frontend/packages/console-app/src/components/dashboards-page/status.ts

```diff
@@ -72,16 +72,16 @@ export const getControlPlaneComponentHealth = (
   return { state: HealthState.ERROR, message: perc.string };
 };
 
-const getWorstStatus = (
+export const getWorstStatus = (
   componentsHealth: SubsystemHealth[],
 ): { state: HealthState; message: string; count: number } => {
-  const withPriority = componentsHealth.map((h) => healthPriority[h.state]);
+  const withPriority = componentsHealth.map((h) => healthPriority?.[h.state] ?? {});
```

Comment:
no need for `healthPriority?.[h.state]`
```suggestion
  const withPriority = componentsHealth.map((h) => healthPriority[h.state] ?? {});
```